Header header

bool in_jig # whether or not the hand is in the calibration jig
